stock_prices/stock_prices.py

```diff
@@ -3,7 +3,23 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  smallest = 0
+  smallestIdx = 0
+  biggest = 0
+  biggestIdx = 0
+
+  for index, value in enumerate(prices):
+    if index == 0:
+      smallest = value
+      smallestIdx = index
+    if value < smallest and biggestIdx == False or not index > biggestIdx:
```

Comment:
very nice check at the end, keeping you from "shorting". 
why `biggestIdx == False` ? Do you ever set it to False anywhere?